### PR TITLE
DISPATCH-1755: Document policy additions to qdstat

### PR DESCRIPTION
**Pull request by ChugR**
@pwright Please review.

### CODE REVIEW

File: docs/man/qdstat.8.adoc [hunk 1]
```diff
@@ -412,6 +412,285 @@ The operational status of this auto link:
...
 The description of the last attach failure that occurred on this auto link.
 
+qdstat --policy
+~~~~~~~~~~~~~~~
+
+Maximum Concurrent Connections::
+The maximum number of concurrent client connections allowed for this router.
+
+Maximum Message Size::
+The maximum size in bytes of AMQP message transfers allowed for this router as messages enter the
```
Comment (pwright) [inline docs/man/qdstat.8.adoc:422]:
```suggestion
The maximum size in bytes of AMQP message transfers allowed for this router.
```
I think the next sentence addresses the 'entering the network' better than the fragment.

File: docs/man/qdstat.8.adoc [hunk 2]
```diff
@@ -412,6 +412,285 @@ The operational status of this auto link:
...
+router network. This limit is applied to transfers over user connections and to transfers to interior
```
Comment (pwright) [inline docs/man/qdstat.8.adoc:423]:
```suggestion
This limit applies to messages entering the router network, that is, to transfers over user connections and to transfers to interior.
```

File: docs/man/qdstat.8.adoc [hunk 3]
```diff
@@ -412,6 +412,285 @@ The operational status of this auto link:
...
+routers from edge routers.
+
+Enable Vhost Policy::
+Enables the router to enforce the connection denials and resource limits defined in the configured
```
Comment (pwright) [inline docs/man/qdstat.8.adoc:427]: defined in the configured vhost policies?

do we need the word 'configured' here?

File: docs/man/qdstat.8.adoc [hunk 4]
```diff
@@ -412,6 +412,285 @@ The operational status of this auto link:
...
+Count of all maxMessageSize policy denials summed across all vhosts.
+
+Total Denials::
+Count of all policy denials for any reason summed across all vhosts.
+
+qdstat --vhosts
+~~~~~~~~~~~~~~~
+
+hostname::
+The hostname of the vhost. This vhost policy will be applied
```
Comment (pwright) [inline docs/man/qdstat.8.adoc:465]:
```suggestion
The hostname of the vhost. This vhost policy applies
```

File: docs/man/qdstat.8.adoc [hunk 5]
```diff
@@ -412,6 +412,285 @@ The operational status of this auto link:
...
+
+maxConnectionsPerUser::
+The maximum number of concurrent client connections allowed for any user.
+
+maxConnectionsPerHost::
+The maximum number of concurrent client connections allowed for any remote host
+(the host from which the client is connecting).
+
+allowUnknownUser::
+Whether unknown users (users who are not members of a defined user group) are allowed to
```
Comment (pwright) [inline docs/man/qdstat.8.adoc:484]: little bit unclear whether this is yes/no true/false or something else

File: docs/man/qdstat.8.adoc [hunk 6]
```diff
@@ -412,6 +412,285 @@ The operational status of this auto link:
...
+The 'hash \#' token matches zero or more tokens. 'asterisk \*' has higher precedence than 'hash \#',
+and exact match has the highest precedence. To specify multiple addresses,
+separate the addresses with either a comma or a space. You can use the
+text string '${user}' in a token to specify an address that contains a
+user's authenticated user name. If you do not specify any addresses then
+users in this group are not allowed to receive messages from any addresses.
+You may specify attributes 'sources' or 'sourcePattern' but not both at
+the same time.
+
+targetPattern::
```
Comment (pwright) [inline docs/man/qdstat.8.adoc:679]: I think this section is good for the man page, but might need some teasing out in docs? wdyt?

**Review by pwright — COMMENTED** [review]
looks good, just some small comments

File: docs/man/qdstat.8.adoc (hunk 2 above)
Comment (ChugR) [reply docs/man/qdstat.8.adoc:423]: agreed

File: docs/man/qdstat.8.adoc (hunk 3 above)
Comment (ChugR) [reply docs/man/qdstat.8.adoc:427]: No, we don't. :-)

File: docs/man/qdstat.8.adoc (hunk 4 above)
Comment (ChugR) [reply docs/man/qdstat.8.adoc:465]: Agreed.

**Pull request closed without merging**